src/types.rs

```diff
@@ -111,7 +109,7 @@ impl Operator {
     }
 }
 
-/// [`Atom`] enum which represents the elementary data unit, the smallest item that is not constructed from smaller parts.
+/// `Atom` enum which represents the elementary data unit, the smallest item that is not constructed from smaller parts.
```

Comment:
How about, `` `Atom`s represent the smallest parts of a term that are not... ``?

Just as you don't need to say "A function to...", you don't need to say things like "A struct that..." or "An enum that...".

Reply:
Oh, bummer - it posted an earlier version of the comment I had here. I meant to ask what you thought of, `` `Atom`s are the parts of a [`TRS`] that are not constructed from smaller parts: [`Variable`]s and [`Operator`]s. ``?